README.md

```diff
@@ -15,23 +15,29 @@ go install github.com/felixge/pprofutils/cmd/...@latest
 
 ## Usage
 
-Convert a pprof file to text:
+Convert a pprof file to folded stack text:
 
-```
+```bash
 pprof2text < ./test-fixtures/pprof.samples.cpu.001.pb.gz > example.txt
 ```
 
-Convert a text file to pprof:
+Convert a folded stack text file to pprof:
 
-```
+```bash
 text2pprof < example.txt > example.pprof
 ```
 
 Warning: Converting from pprof to text is lossy. Only the first sample type will be converted, file names, lines, labels, and more will be dropped. Patches to make things less lossy would be welcome, but please open an issue first to discuss.
 
-Create a delta profile that contains the difference `heap-b.pprof - heap-a.pprof`:
+Convert a Linux `perf.data` profile to `pprof`, via Brendan Gregg's [`stackcollapse-perf.pl`](https://github.com/brendangregg/FlameGraph/blob/master/stackcollapse-perf.pl) script:
 
+```bash
+stackcollapse-perf.pl < perf.data | text2pprof > perf.pprof
```

Comment:
NIT: This makes it seem like `stackcollapse-perf.pl` can process a binary perf data file, which is not the case. `stackcollapse-perf.pl` can process the output of `perf script`. So maybe this snippet should read:

```suggestion
perf script | stackcollapse-perf.pl | text2pprof > perf.pprof
```